task_03.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""This is week8 synthesizing task_3"""
+
+import decimal
+
+NAME = raw_input('What is your Name? ')
+PRINCIPAL = int(raw_input('What is the Ammount of your Principal? '))
+YEARS = int(raw_input('For how many years is Loan borrowed? '))
+PREQ = raw_input('Are you Pre-qualified? ').lower()
```

Comment:
This could never be `False` with the values `yes|y|no|n` because they're all strings which are non-empty and thus always `True` when evaluated.